css/main.css

```diff
@@ -0,0 +1,617 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+hr {
+  width: 80vw;
+}
+
+header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px;
+  background-color: #3c3a39;
+  color: white;
+}
```

Comment:
[OPTIONAL] I found a little bug that you can fix it easily by changing the position of the menu. 
After clicking on one of the menu options, the menu is still at the top of the page, so I have to scroll up to find it.
Check the `position` CSS value of your menu